src/stylesheets/utility/_headings.scss

```diff
@@ -44,3 +44,18 @@
     }
   }
 }
+
+.lg-title {
+  color: color(emerald-green);
+  font-family: $serif;
+  font-size: rem(30);
+  font-weight: lighter;
```

Comment:
My initial hunch was that font-weight keywords other than normal and bold, might not have consistent browser support, so I did a search. Turns out that's not correct, BUT keyword values are dependent on what's available in the font family. https://css-tricks.com/almanac/properties/f/font-weight/#article-header-id-0 Just something to keep in mind with future work.

Reply:
Oh interesting. After playing with it some more there really is no `lighter` for this font, just `normal` or `bold` so I kept `normal`.